cli_reference/kn-cli-tools.adoc

```diff
@@ -0,0 +1,44 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="kn-cli-tools"]
+= Knative CLI (`kn`) for use with OpenShift Serverless
+:context: kn-cli-tools
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+`kn` enables simple interaction with Knative components on {product-title}.
+
+You can enable Knative on {product-title} by installing {ServerlessProductName}. For more information, see the documentation on xref:../serverless/serverless-getting-started.adoc#serverless-getting-started[Getting started with {ServerlessProductName}].
+
+[NOTE]
+====
+{ServerlessProductName} cannot be installed using the `kn` CLI. A cluster administrator must install the {ServerlessOperatorName} and set up the Knative components, as described in the _Serverless applications_ documentation for {product-title}.
+====
+
+== Key features
+`kn` is designed to make serverless computing tasks simple and concise, and has the following key features:
```

Comment:
I'd break up this sentence. "Key features include:"

Reply:
Be sure each list item ends with a period and is a complete sentence. Try to start with a verb. I'll add some suggestions inline.